Changelog.rst

```diff
@@ -20,6 +20,8 @@ version 3.15.0
   (https://github.com/NCAS-CMS/cf-python/issues/626)
 * Removed benign UserWarning from `cf.Field.percentile`
   (https://github.com/NCAS-CMS/cf-python/issues/619)
+* Handled the renaming of the ESMF Python interface from `ESMF` to
+  `esmpy` at version 8.4.0. Both module names are accepted for now.
 * Changed dependency: ``1.10.1.0<=cfdm<1.10.2.0``
```

Comment:
Just spotted a version change in Changelog:
```suggestion
* Changed dependency: ``1.10.1.0<=cfdm<1.10.2.0``
* Changed dependency: ``8.0.0<=esmpy``
```

Reply:
Good spot. Do you mind re-suggesting (i.e. with the suggestions feature) without the trailing whitespace on the first line? That way I don't have to re-commit to remove it...

Reply:
Oh, fair - copying over the suggestion showed/shows there was one character of it but maybe that's just a facet of the copy highlighting feature. Accepting the suggestion now.